go/oasis-test-runner/env/env.go

```diff
@@ -192,8 +192,11 @@ func (env *Env) Cleanup() {
 	}
 
 	// Tear down this environment's commands.
-	for _, v := range env.cleanupCmds {
-		v.termOrKill()
+	for _, m := range env.cleanupCmds {
+		go m.termOrKill()
```

Comment:
Are we allowed to terminate these services concurrently?

Reply:
This looks fine as it is just sending the signals in parallel.